### PR TITLE
Updated Tracker Icons

### PR DESCRIPTION
**Pull request by NWBx01**
Added SlimeVR-specific tracker icons to be used and removed old icons which were unused.

I'm unfamiliar with where exactly the icons are being loaded from currently, so although I changed the paths for the tracker icons, I believe "placeholder_tracker_status.png" will still be loaded.

### CODE REVIEW

**Comment by NWBx01** [comment]
> Why some of the files have suffix `b4bfb144`?
I was following in the convention of icon naming as others use. The **b4bfb144** portion designates that it's in the standard SteamVR blue-green gradient, whereas those without are pea green, with the exception of "tracker_status_off.png", which is a dark gray.

**Review by Eirenliel — APPROVED** [review]

**Comment by Eirenliel** [comment]
> I'm unfamiliar with where exactly the icons are being loaded from currently, so although I changed the paths for the tracker icons, I believe "placeholder_tracker_status.png" will still be loaded.

we should check this and fix it, otherwise this PR does nothing and can't be accepted. at the very least we need to create a separate issue to fix it. i love the icons and i want them 🥺

**Comment by ButterscotchV** [comment]
Just tested the driver build from CI, it looks like it works fine to me :3
image

**Comment by ButterscotchV** [comment]
Also just noticed that we're actually missing the icon for "Not Detected" (when a tracker position is disabled in SlimeVR)
Here I disabled feet, you can see what the result is:
image
image

File: src/TrackerDevice.cpp [hunk 1]
```diff
@@ -141,15 +141,17 @@ vr::EVRInitError SlimeVRDriver::TrackerDevice::Activate(uint32_t unObjectId)
...
-    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceSearching_String, "{slimevr}/icons/tracker_not_ready.png");
-    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceSearchingAlert_String, "{slimevr}/icons/tracker_not_ready.png");
-    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceReadyAlert_String, "{slimevr}/icons/tracker_not_ready.png");
-    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceNotReady_String, "{slimevr}/icons/tracker_not_ready.png");
-    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceStandby_String, "{slimevr}/icons/tracker_not_ready.png");
-    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceAlertLow_String, "{slimevr}/icons/tracker_not_ready.png");
+    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceReady_String, "{slimevr}/icons/tracker_status_ready.b4bfb144.png");
+
+    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceOff_String, "{slimevr}/icons/tracker_status_off");
```
Comment (kitlith) [inline src/TrackerDevice.cpp:146]: missing .png extension

Comment (kitlith) [reply src/TrackerDevice.cpp:146]: @ButterscotchV if you add the missing .png extension here, does that fix it?

Comment (ButterscotchV) [reply src/TrackerDevice.cpp:146]: I can't compile it rn but here's adding that as a suggestion :P
```suggestion
    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceOff_String, "{slimevr}/icons/tracker_status_off.png");
```

Comment (kitlith) [reply src/TrackerDevice.cpp]: good call, i applied the suggestion. it's going through CI now.

**Review by ButterscotchV — APPROVED** [review]
Tested from CI artifact and seems to be working great

File: src/TrackerDevice.cpp [hunk 2]
```diff
@@ -141,15 +141,17 @@ vr::EVRInitError SlimeVRDriver::TrackerDevice::Activate(uint32_t unObjectId)
...
+    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceSearching_String, "{slimevr}/icons/tracker_status_ready.b4bfb144.png");
+    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceSearchingAlert_String, "{slimevr}/icons/tracker_status_ready_alert.b4bfb144.png");
+    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceReadyAlert_String, "{slimevr}/icons/tracker_status_ready_alert.b4bfb144.png");
+    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceNotReady_String, "{slimevr}/icons/tracker_status_off.png");
+    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceStandby_String, "{slimevr}/icons/tracker_status_standby.b4bfb144.png");
+    GetDriver()->GetProperties()->SetStringProperty(props, vr::Prop_NamedIconPathDeviceAlertLow_String, "{slimevr}/icons/tracker_status_ready_low.b4bfb144.png");
+
+    // Note: icons not currently being loaded. Instead, "placeholder_tracker_status.png" is used.
```
[the remark this answers is not in the document]
Comment (kitlith) [reply src/TrackerDevice.cpp:154]: If this is actually working, then we should probably remove the comment here.

**Review by ButterscotchV — APPROVED** [review]

**Review by kitlith — APPROVED** [review]

**Pull request merged**